pkgs/development/interpreters/python/passthrufun.nix

```
@@ -91,9 +91,9 @@ in rec {
     pythonAtLeast = lib.versionAtLeast pythonVersion;
     pythonOlder = lib.versionOlder pythonVersion;
     inherit hasDistutilsCxxPatch;
-    # TODO: deprecate
-    # Not done immediately because its likely used outside Nixpkgs.
-    pythonForBuild = pythonOnBuildForHost_overridden;
+    pythonForBuild =
```

```suggestion
    # Remove after 24.11 is released.
    pythonForBuild =
```

3 releases should be enough I think

---

Only used in ~66 places https://github.com/search?q=%2Fpython3.pythonForBuild%2F%20lang%3Anix%20NOT%20is%3Afork&type=code

---

> Only used in ~66 places https://github.com/search?q=%2Fpython3.pythonForBuild%2F%20lang%3Anix%20NOT%20is%3Afork&type=code

There are a few more uses that aren't caught by that search string (like `inherit (python3) pythonForBuild`).  This PR should fix all of them.